<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_SuperUsuario</class>
 <widget class="QDialog" name="Dialog_SuperUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_Usuario">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="crear_usuario">
      <attribute name="title">
       <string>Crear Usuario</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_nuevo_usuario">
         <property name="title">
          <string>Nuevo Usuario</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit_crear_usuario_nombre">
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Nombre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_crear_usuario_apellido">
            <property name="placeholderText">
             <string>Apellido</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_crear_usuario_password">
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Contraseña</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_asigna_empresa"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_asignar_rol">
         <property name="title">
          <string>Asignar Rol</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="comboBox_asignar_rol"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_crear_usuario">
         <property name="text">
          <string>Crear Usuario</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="editar_usuario">
      <attribute name="title">
       <string>Editar Usuario</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_busca_usuario">
         <property name="title">
          <string>Buscar Usuario</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEdit_editar_usuario_buscar"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_editar_usuario_buscar">
            <property name="text">
             <string>Buscar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="listView_edita_usuario"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_editar_usuario">
         <property name="title">
          <string>Editar Usuario</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLineEdit" name="lineEdit_editar_nombre_usuario">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Nombre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_editar_apellido_usuario">
            <property name="placeholderText">
             <string>Apellido</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_editar_password_usuario">
            <property name="placeholderText">
             <string>Contraseña</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_rol_editar_usuario">
            <property name="text">
             <string>Rol</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_editar_rol_usuario">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_empresa_editar_usuario">
            <property name="text">
             <string>Empresa</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_editar_empresa_usuario"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_editar_guardar_usuario">
            <property name="text">
             <string>Guardar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="eliminar_usuario">
      <attribute name="title">
       <string>Eliminar Usuario</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_busca_usuario_2">
         <property name="title">
          <string>Eliminar Usuario</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="lineEdit_eliminar_usuario_buscar"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_eliminar_usuario_buscar">
            <property name="text">
             <string>Buscar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_eliminar_usuario"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_eliminar_usuario">
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ver_usuario">
      <attribute name="title">
       <string>Ver Usuario</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QGroupBox" name="groupBox_busca_usuario_3">
         <property name="title">
          <string>Buscar Usuario</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLineEdit" name="lineEdit_ver_usuario_buscar"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ver_usuario_buscar">
            <property name="text">
             <string>Buscar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget__ver_usuario"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
